scripts/convert_csv_to_json.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import pandas as pd
+import sys
+import json
+
+
+def generate_json_files(version):
+
+    source_directory = './source-data/'
+    output_directory = './data/' + version + '/'
+
+    for source_file_name in os.listdir(source_directory):
+        if os.path.isfile(os.path.join(source_directory, source_file_name)):
```

Comment:
You have an 'if' to check for file, but no output if there isn't any

Reply:
Not sure if anything would ever not be a regular file in the context of running this script but I'll add a message to that effect if it isn't.